src/main/java/games/strategy/engine/lobby/server/db/HashedPassword.java

```diff
@@ -15,6 +19,14 @@ public HashedPassword(final String hashedPassword) {
    * Returns true if the hashed password looks like it could be a hash.
    */
   public boolean isValidSyntax() {
+    return isMd5Crypted() || isBcrypted();
+  }
+
+  public boolean isBcrypted() {
```

Comment:
Checking the syntax of the hash is not my favorite way to do this, but okay. This method in my mind is more of a feature flag as well than anything else, it could be just as well called `isLatestEncryption()` for example. I don't have strong objections to this, it is temporary after all.